organization/_posts/2023/2023-03-02-coordination.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "HSF Coordination Meeting #245, 2 March 2023"
+layout: plain_toc
+---
+
+## Attending
+
+Present/Contributing: Graeme Stewart, Benedikt Hegner, Phil Ilten, Michel Jouvin, Krzysztof Genser, Patrick Gartung, Claire Antel, Torre Wenaus, Stefan Roiser, Alexander Held, Paul Laycock, Christina Agapopoulou
```

Comment:
I wonder if this is the complete list? I am usually taking screenshots of the attendees because a lot of people don't write their names down.

Reply:
Probably not. I asked Michel to take notes, but I guess that was not part of it.